lib/charms/loki_k8s/v0/loki.py

```diff
@@ -115,22 +375,98 @@ def __init__(self, charm: CharmBase, relation_name: str):
                 `self` in the instantiating class.
             relation_ name: a string name of the relation between `charm` and
                 the Loki charmed service.
+            alert_rules_path: an optional path for the location of alert rules
+                files.  Defaults to "src/loki_alert_rules" at the top level
+                of the charm repository.
         """
         super().__init__(charm, relation_name)
         self._stored.set_default(loki_push_api=None)
         self._charm = charm
         self._relation_name = relation_name
+        self._ALERT_RULES_PATH = alert_rules_path
```

Comment:
why all caps?

Reply:
Just because inside the object is a constant